.github/workflows/push.yaml

```diff
@@ -0,0 +1,206 @@
+
+      - name: Generate image repository path
+        run: |
+          echo IMAGE_REPOSITORY=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ${{ env.DOCKER_FILE_PATH  }}
+          pull: true
+          push: true
+          build-args: BUILD_PARAMETERS=${{ env.BUILD_PARAMETERS }}
+          cache-to: type=inline
+          tags: |
+            ${{ env.IMAGE_REPOSITORY }}:${{ steps.generate_tag.outputs.new_tag }}
+          labels: |
+            org.opencontainers.image.source=${{ github.event.repository.clone_url }}
+            org.opencontainers.image.created=${{ steps.prep.outputs.created }}
+            org.opencontainers.image.revision=${{ github.sha }}
+      ##############################
+      ## Add steps to generate required artifacts for a release here(helm chart, operator manifest etc.)
+      ##############################
+
+      # Generate tag for operator without "v"
+      - name: Generate Operator Tag
+        id: generate_operator_tag
+        uses: anothrNick/github-tag-action@1.26.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.STAKATER_GITHUB_TOKEN }}
+          WITH_V: false
+          DEFAULT_BUMP: patch
+          DRY_RUN: true
+
+      # Install operator-sdk
+      - name: Install operator-sdk
+        env:
+          OPERATOR_SDK_VERSION: ${{ env.OPERATOR_SDK_VERSION }}
+        run: |
+          curl -fL -o /tmp/operator-sdk "https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk-v${OPERATOR_SDK_VERSION}-x86_64-linux-gnu"
+          sudo install /tmp/operator-sdk /usr/local/bin && rm -f /tmp/operator-sdk
+          operator-sdk version
+          operator-sdk version | grep -q "${OPERATOR_SDK_VERSION}"
+
+      # Install Kustomize
+      - uses: imranismail/setup-kustomize@v1
+        with:
+          kustomize-version: ${{ env.KUSTOMIZE_VERSION }}
+
+      # - name: Generate Bundle
+      #   env:
+      #     VERSION: ${{ steps.generate_operator_tag.outputs.new_tag }}
+      #   run: make bundle
+
+      # Update chart tag to the latest semver tag
+      - name: Update Chart Version
+        env:
+          VERSION: ${{ steps.generate_operator_tag.outputs.new_tag }}
+        run: make bump-chart
+      
+      # - name: Update Chart CRDs
+      #   run: make generate-crds
+        
+      # Setting up helm binary
+      - uses: azure/setup-helm@v1
+
+      # Publish helm chart to
+      - name: Publish Helm chart
+        run: |
+          helm plugin install https://github.com/chartmuseum/helm-push.git
+          helm package deployments/kubernetes/chart/reloader/* --destination ./packaged-chart
+          helm push ./packaged-chart/*.tgz ${{ env.HELM_REGISTRY_URL }}
+          # curl ${{ env.HELM_REGISTRY_URL }} --upload-file ./packaged-chart/*.tgz
```

Comment:
There is no point in writing a step manually for this. Please, use the action from [here](https://github.com/stakater/jira-service-desk-operator/blob/master/.github/workflows/push.yml#L154) instead.